troposphere/static/js/components/common/InstanceDetail.jsx

```diff
@@ -59,7 +54,7 @@ const InstanceDetail = React.createClass({
     },
 
     renderActiveInstance: function(instance) {
-        let { InstanceStore, ProjectStore } = this.props.subscriptions;
+        let { ProjectStore } = this.props.subscriptions;
```

Comment:
I still think that using "string"s is a reasonable compromise - but this definitely shows how you miss out on the static analysis.